src/zcbenchmarks.cpp

```diff
@@ -420,3 +524,15 @@ double benchmark_sendtoaddress(CAmount amount)
     return timer_stop(tv_start);
 }
 
+double benchmark_loadwallet()
+{
+    Shutdownpartial();
+    struct timeval tv_start;
+    bool fFirstRunRet=true;
+    timer_start(tv_start);
+    pwalletMain = new CWallet("wallet.dat");
+    DBErrors nLoadWalletRet = pwalletMain->LoadWallet(fFirstRunRet);
+    return timer_stop(tv_start);
```

Comment:
This needs to go at the end of the function, otherwise the re-registrations aren't doing anything.

Reply:
oopss. moved return to end (Though stopped timer at same place)